Source/SwiftLintFramework/Rules/SortedImportsRule.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  SortedImportsRule.swift
+//  SwiftLint
+//
+//  Created by Scott Berrevoets on 12/15/16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct SortedImportsRule: ConfigurationProviderRule, OptInRule {
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "sorted_imports",
+        name: "Sorted Imports",
+        description: "Imports should be sorted.",
+        nonTriggeringExamples: [
+            "import AAA\nimport BBB\nimport CCC\nimport DDD"
+        ],
+        triggeringExamples: [
+            "import AAA\nimport ZZZ\nimport BBB\nimport CCC",
```

Comment:
You should be inserting the `↓` character where you expect the violation to be reported.